app/controllers/spree/api/line_items_controller_decorator.rb

```diff
@@ -1 +1,5 @@
-Spree::Api::LineItemsController.include SpreeMultiDomain::CreateLineItemSupport
+unless SpreeMultiDomain::Config.ignore_api_controllers?
+  Spree::Api::LineItemsController.include(
+    SpreeMultiDomain::CreateLineItemSupport
```

Comment:
Any reason why these includes have been put on to new lines? I think it makes the code less readable. Same for the others.